File: T8. DATA TYPES AND VARIABLES - EXERCISES (Java)/src/P06_StringsAndObjects.java

public class P06_StringsAndObjects {
    public static void main(String[] args) {

        String word1 = "Hello";
        String word2 = "World";

        Object sumWords = word1 + " " + word2;

        String finalResult = String.valueOf(sumWords);

        System.out.println(finalResult);
    }
}

File: T8. DATA TYPES AND VARIABLES - EXERCISES/P06.StringsAndObjects.cs
using System;

namespace P06.StringsAndObjects
{
    class Program
    {
        static void Main(string[] args)
        {
            string word1 = "Hello";
            string word2 = "World";

            object sumWord = word1 + " " + word2;

            string final = (string)sumWord;

            Console.WriteLine(final);
        }
    }
}
